lib/venue.rb

```diff
@@ -1,2 +1,19 @@
 class Venue
+  attr_reader :name, :capacity
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def patrons
+    []
+  end
+
+  def add_patron(add_patron)
+    @patrons = add_patron
```

Comment:
What data structure is `@patrons`? How do you add something to an array?

https://ruby-doc.org/core-2.5.0/Array.html#:~:text=Adding%20Items%20to,4%2C%205%2C%206%5D